### DISCUSSION TITLE
Help me, pls!

### DISCUSSION THREAD

**Discussion by ScrepY** [codewars/codewars.com#2723] [Q&A]
I can't solve this problem, please help me :( (https://www.codewars.com/kata/62a8c5d9936a4a005c0b6a32/javascript)

**Comment by razetime** [comment]
what have you written so far?

**Reply by shokhrukh1212** [reply]
let me see what you wrote and then I can help you, it is not as difficult as you thought

**Reply by hobovsky** [reply]
I deleted a post with a spoiler.

Please don't post spoilers. 
You could get a better help at `#help-solve` channel of Codewars Discord.

**Comment by saymowan** [comment]
> I can't solve this problem, please help me :( (https://www.codewars.com/kata/62a8c5d9936a4a005c0b6a32/javascript)

Share your insights with us.
